### PR TITLE
adding our modified BG snark construction paper

### PR DESCRIPTION
**Pull request by vanishreerao**
Adding the modified BG snark paper here. And corresponding edits to Link.re. Please check my edits in Link.re in particular. Thanks.

### CODE REVIEW

File: frontend/website/src/Links.re [hunk 1]
```diff
@@ -46,6 +46,11 @@ module Static = {
     link: Cdn.url("/static/coda-whitepaper-05-10-2018-0.pdf"),
   };
 
+  let modifiedsnark = () => {
+    name: ("Read the zkSNARK We Are Using", "modified-BG-snark"),
```
Comment (enolan) [inline frontend/website/src/Links.re]: Grammar: should be "Read about the zkSNARKs we're using". Or maybe the "Read about the zkSNARK construction we're using".

Comment (vanishreerao) [reply frontend/website/src/Links.re]: Done.

File: frontend/website/src/Links.re [hunk 2]
```diff
@@ -46,6 +46,11 @@ module Static = {
     link: Cdn.url("/static/coda-whitepaper-05-10-2018-0.pdf"),
   };
 
+  let modifiedsnark = () => {
+    name: ("Read About the zkSNARK construction We're Using", "modified-BG-snark"),
```
Comment (bkase) [inline frontend/website/src/Links.re]: We should make sure we don't over/under capitalize here "Read about the zkSNARK construction we're using"

Comment (vanishreerao) [reply frontend/website/src/Links.re]: Done.

**Pull request merged**